packages/nimble-components/src/utilities/wait-until-custom-elements-defined-async.ts

```diff
@@ -0,0 +1,8 @@
+export async function waitUntilCustomElementsDefinedAsync(
```

Comment:
Note: need to update rich-text base class to use this after merging

Reply:
Also in `RichTextMentionListBox`, once that goes in.

Reply:
updated the rich-text base class and doesn't look like the RichTextMentionListBox used that pattern.